api/http/server.go

```diff
@@ -732,15 +737,33 @@ func (s *Server) HandleGetFeedRaw(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	data, err := s.api.RetrieveFeedUpdate(r.Context(), ref)
+	timeout := r.URL.Query().Get("timeout")
```

Comment:
Please consider putting this in a header instead of a query string. In my view this is a directive to the web server than it is an instruction to the actual retrieval layer.